helpers/httpApi.js

```diff
@@ -140,6 +140,42 @@ var middleware = {
 	attachResponseHeaders: function (getHeaders, req, res, next) {
 		res.set(getHeaders());
 		return next();
+	},
+
+	/**
+	 * Lookup cache, and reply with cached response if it's a hit.
+	 * If it's a miss, forward the request but cache the response if it's a success.
+	 * @param {Object} req
+	 * @param {Object} res
+	 * @param {Function} next
+	 */
+	useCache: function (logger, cache, req, res, next) {
+		if (cache.isReady()) {
```

Comment:
Can we skip one nesting level? For example:
`if (!cache.isReady()) { next(); }`